docs/local-deployment/local-directory/overview.mdx

```diff
@@ -6,9 +6,9 @@ description: Deploy the Aserto Directory locally
 
 # Local Directory Overview
 
-The Aserto Directory can be run as a standalone container, backed by a Postgres DB.
+For advanced scenarios, the Aserto Directory can be deployed locally as a standalone container, backed by a Postgres DB, along with a local management console, and an Edge Authorizer. When running locally, there is no dependency on externally hosted components nor will your local Asert environment share any data externally. Optionally, you can configure the Authorizer to pull down container images from an OCI v2 Registry (such as Github), and you can import data from an IDP (Auth0) using the `ds-load` CLI. 
```

Comment:
"your local Aserto environment" 
"you can configure the Edge Authorizer...."
"from an IDP (Auth0, Azure AD, etc.)